app/styles/input.css

```diff
@@ -1,13 +1,17 @@
 /*
  * noVNC general input element CSS
- * Copyright (C) 2022 The noVNC authors
+ * Copyright (C) 2025 The noVNC authors
  * noVNC is licensed under the MPL 2.0 (see LICENSE.txt)
  * This file is licensed under the 2-Clause BSD license (see LICENSE.txt).
  */
 
 /*
  * Common for all inputs
  */
+:root {
+    --input-xpadding: 1em;
```

Comment:
I would vote against a variable for this, as it is so few places that need it.

Reply:
It is used in 6 places, in both input.css and base.css. I'll keep it for now.